.changeset/shaggy-cats-film.md

```diff
@@ -0,0 +1,11 @@
+---
+"astro": minor
+---
+
+Extends the acceptable cookie options when deleting a cookie, to the following attribute:
+
+- `domain`
+- `path`
+- `httpOnly` *(Added)*
+- `sameSite` *(Added)*
+- `secure` *(Added)*
```

Comment:
```suggestion
Adds the `httpOnly`, `sameSite` ans `secure` options when deleting a cookie
```

Reply:
Nicely done @florian-lefebvre ! Just a typo for "and" missing Oxford comma in there, but otherwise, I approve this! :purple_heart: 

>Adds the `httpOnly`, `sameSite`, and `secure` options when deleting a cookie

Reply:
Updated! Didn't know about this Oxford thing, we don't put a coma there in French